_posts/2016-05-27-floss.md

```diff
@@ -0,0 +1,100 @@
+
+However, in the 70's and early 80's, the evolving software industry began to directly sell software, which usually was in the form of binary copies, thereby preventing users to reverse engineer the software. In 1980, the copyright law in the States was extended to software programs.
+
+In 1983, [Richard Stallman](“https://stallman.org”) at the MIT AI Laboratory, announced the *GNU Project*. Along with the GNU OS development, the *Free Software Foundation a.k.a. FSF* was founded in 1985. The [GNU Manifesto]("https://en.wikipedia.org/wiki/GNU_Manifesto") contained the GNU philosophy of Free Software definition and "copyleft" ideas.
+
+The Linux kernel was released as a freely modifiable open source software in 1991, which was relicensed under the GNU GPL in 1992.
+FreeBSD and NetBSD were released as free software in 1993. In 1995, the [Apache Web Server](“https://httpd.apache.org”) (A patchy web server, there is sure history behind its name :P) was released.
+
+Then came a revolutionary moment in 1998. The Netscape Corporation released its famous Netscape Communicator Suite as free software. (I am sure you guys must have heard Firefox and ThunderBird. Yeah, that is the descendant of the Netscape; however, another story, another time).
+
+Netscape saw business potential in collaborating and sharing source code, however the FSF and its ideals were not not appealing enough to them. Hence, they decided to rebrand the movement; coining the famous "open source" and the [Open Source Initiative](%22https://opensource.org/%22) was established in February, 1998. In 2001, the European Commission introduced the term FLOSS, which has been in both popular and official use since 2004.<sup>[2]</sup>
+
+
+## Licensing : An important part
+
+Since the establishment of the GNU Project, there have been ideas floating around for an explicit open source license. The GNU General Public License v1.0 was thus released in 1989, with v2.0 coming in 1991.
+
+Various popular licenses pertaining to Free software, as approved by the OSI are: 	<sup>[3]</sup>
+
+1. Apache License 2.0
+
+2. BSD 3-Clause "New" or "Revised" license
+
+3. BSD 2-Clause "Simplified" or "FreeBSD" license
+
+4. GNU General Public License (GPL)
+
+5. GNU Library or "Lesser" General Public License (LGPL)
+
+6. MIT license
+
+7. Mozilla Public License 2.0
+
+8. Common Development and Distribution License
+
+9. Eclipse Public License
+
+Check out these licenses and their full texts at [the OSI website]("https://opensource.org/licenses”).
+
+## GNU/Linux
+
+One can't write an article on FLOSS, and skip out GNU/Linux. There is quite a lot of history surrounding the most popular OSS Linux kernel, and it sure is a favourite amongst developers and the community alike. 
+There are a lot of Operating systems (referred to as linux distributions) based on the linux kernel, popular ones being 
+##Ubuntu##
```

Comment:
Last thing, you forgot this I guess :stuck_out_tongue: And please add a newline before Ubuntu as well.